crawler/file_processing.py

```diff
@@ -597,7 +603,7 @@ def insert_samples_from_docs_into_mongo_db(self, docs_to_insert: List[Dict[str,
             logger.exception(e)
             return []
 
-    def insert_samples_from_docs_into_mlwh(self, docs_to_insert) -> bool:
+    def insert_samples_from_docs_into_mlwh(self, docs_to_insert):
```

Comment:
why remove the return type annotation?

Reply:
This was part of my MonkeyType testing, to see if it would pick this up.

Reply:
Still in progres...

Reply:
Can leave it for now as a "gardening" task along with another story.